docs/providers/aws/events/kafka.md

```diff
@@ -60,7 +117,7 @@ functions:
             - abc2.xyz.com:9092
 ```
 
-## Enabling and disabling Kafka event
+## Example: Enabling and disabling Kafka event trigger
```

Comment:
I think we can get rid of "example: " prefix as we don't do that across documentation so far and it would be good to stay consistent with formatting